main.js

```diff
@@ -0,0 +1,48 @@
+
+snakewaterMontana = {
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velocirapto"
+};
+
+var guestOfHonor = snakewaterMontana.paleontologist;
+var cleverGirl = snakewaterMontana.specimen;
+console.log (cleverGirl);
+
+nicaragua = {
+    depth: "200 meters",
+    annualBudget: 1500000,
+    specimens: [
+      "Tyrannosaurus Rex",
+      "Stegosaurous",
+      "Triceratops",
+      "Velociraptor"
+    ]
+  }
+ var specimens = nicaraguanSpecimens 
```

Comment:
you cant assign an undefined variable to another variable anytime you want to use a variable or store something you need to define it first with `var` keyword, in this part we need to store the array of specimens into a variable called nicaraguanSpecimens to do that we define a variable first and name it ` var nicaraguanSpecimens  ` then we access the specimens  array which inside nicaragua object the final solution will be something like this ` var nicaraguanSpecimens  =  nicaragua.specimens: `